nrf_802154/driver/include/nrf_802154.h

```diff
@@ -1147,7 +1177,7 @@ bool nrf_802154_pan_coord_get(void);
  *
  * When calling @ref nrf_802154_ack_data_pending_bit_should_be_set, one of several algorithms
  * for source address matching will be chosen. To ensure a specific algorithm is selected,
- * call this function before @ref rf_802154_ack_data_pending_bit_should_be_set.
+ * call this function before @ref nrf_802154_ack_data_pending_bit_should_be_set.
```

Comment:
`nrf_802154_ack_data_pending_bit_should_be_set` is not-public API as well. Should this entire paragraph even exist in this case? This isn't something useful to the user.